source/DG.Core/Scanners/ConfigJSONFileScanner.cs

```diff
@@ -0,0 +1,110 @@
+﻿namespace DG.Core.Scanners
+{
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using Microsoft.Extensions.Configuration;
+
+    public class ConfigJSONFileScanner : IConfigScanner
+    {
+        private readonly string configPath;
+
+        public ConfigJSONFileScanner(string pathToConfig)
+        {
+            this.configPath = pathToConfig;
+        }
+
+        public IDictionary<string, object> GetKeyValuesFromAllSections()
+        {
+            var configurationSections = this.GetBaseConfigSections();
+            IDictionary<string, object> sectionValues = new Dictionary<string, object>();
+            foreach (ConfigurationSection configurationSection in configurationSections)
```

Comment:
We can increase readability by using `var`:
`var sectionValues = new Dictionary<string, object>();`
`foreach (var configurationSection in configurationSections)`

Reply:
I guess we can try to use IOptions<T> here https://blog.zwezdin.com/2016/configuring-aspnet-core-apps-with-ioptions/

Reply:
I haven't find good decision to use IOptions in DG.Core project component.